tasks/installer-setup.yml

```diff
@@ -75,7 +75,7 @@
 - name: Disable agent install if owned by installer
   set_fact:
     agent_datadog_skip_install: true
-  when: datadog_installer_owns_agent.rc == 0
+  when: not datadog_installer_bootstrap_result.failed and datadog_installer_owns_agent.rc == 0
```

Comment:
❓ Question :
Just by lack of context, what's supposed to happen when the bootstrap fail aside from crashing ? Or would it be safer to crash ? to notify the failure ?

Reply:
We carry on in order to report telemetry afterward, and only then notify of the error.
There might be some more elegant ways to do so though